apis/r/R/roxygen.R

```diff
@@ -169,3 +169,133 @@ rd_soma_field <- function(field) {
     )
   ))
 }
+
+rd_outgest_index <- function(type = 'v3', axis = 'obs') {
```

Comment:
Please spell out whatever rd is standing for

You do save yourself a few keystrokes typing it out here, as a one-time cost, but the added clarity to every reader forever after will be a net time savings

Reply:
Rd stands for "R documentation", but is referred to as "Rd" in all manuals (such as ["Writing R Extensions"](https://cran.r-project.org/doc/manuals/R-exts.html#Rd-format) and ["Parsing Rd Files"](https://developer.r-project.org/parseRd.pdf)) and help pages (such as [`tools::parse_Rd()`](https://stat.ethz.ch/R-manual/R-devel/library/tools/html/parse_Rd.html)) within R itself

Reply:
Then please add a top-of-function comment that this is an R-documentation generator.

This is quite obvious to the R-jedi (you being one) but this is a polyglottal project, with polyglottal developers as well as polyglottal users. Please just add an explanatory comment for the rest of us.